agents/human_agent.py

```diff
@@ -8,8 +9,33 @@ class HumanAgent(Agent):
     the user choose the action.
     TODO - implement everything!
     """
+
     def __init__(self, name):
         super(HumanAgent, self).__init__(name)
 
     def choose_action(self, env):
-        pass
\ No newline at end of file
+        # create list of available moves
+        available_moves = env.available_moves(self)
+
+        # check if the list are not empty
+        if available_moves:
+
+            # action of the player
+            while True:
+                print("Please choose your action from the list below and press ENTER:")
+                print(" # | col")
+                print("---------")
+                for act in enumerate(available_moves):
+                    print(" {0} |  {1}".format(act[0], act[1]))
+                move = input()
+
+                # check input
+                if move == 'q':
+                    sys.exit(0)
+                try:
+                    return available_moves[int(move)]
+                except:
```

Comment:
As discussed on skype - add exception cases for `move` not being castable to int (e.g. if `move` is `"a"`) or if index error.